src/AnalyzeView/ExifParser.cc

```diff
@@ -110,7 +101,7 @@ double readTime(const QByteArray &buf)
     return (tagTime.toMSecsSinceEpoch() / 1000.0);
 }
 
-bool write(QByteArray& buf, const GeoTagWorker::cameraFeedbackPacket& geotag)
+bool write(QByteArray &buf, const GeoTagWorker::cameraFeedbackPacket &geotag)
```

Comment:
FYI: Most of the code uses the `foo& bar` style instead of `foo &bar`. Just pointing out, it's really fine either way. I've always thought the modifier `&/*` should go closer to the type since it's more related to that. I know weird. Just a quirk I have.